content/ar/docs/tutorials/hello-minikube.md

```diff
@@ -0,0 +1,303 @@
+---
+title: مرحبًا، مينيكيوب
+content_type: tutorial
+weight: 5
+menu:
+  main:
+    title: "ابدأ الآن"
+    weight: 10
+    post: >
+       <p>مستعد للبدء؟ قم ببناء عنقود كوبيرنيتيس بسيط لتشغيل تطبيق تجريبي .</p>
+card:
+  name: tutorials
+  weight: 10
+---
+
+<!-- overview -->
+
+يعرض هذا الدليل كيفية تشغيل تطبيق تجريبي على كوبيرنيتيس باستخدام مينيكيوب.
+يقدم الدليل صورة حاوية  لإنجينكس (Nginx) لإعادة توجيه جميع الطلبات على الشبكة.
+
+## {{% heading "objectives" %}}
+
+* نشر تطبيق تجريبي على عنقود مينيكيوب.
+* تشغيل التطبيق.
+* عرض سجلات التطبيق.
+
+## {{% heading "prerequisites" %}}
+
+يفترض هذا البرنامج التعليمي أنك قمت بإعداد `مينيكيوب` مسبقاً.
+راجع تعليمات الإعداد في __الخطوة 1__ من [بدء مينيكيوب](https://minikube.sigs.k8s.io/ar/docs/start/) للحصول على تعليمات التثبيت.
+
+{{< note >}}
+قم بتنفيذ التعليمات فقط في __الخطوة 1، التثبيت__. الباقي مغطى في هذه الصفحة.
+{{< /note >}}
+
+كما يتعين عليك تثبيت `kubectl`.
+يرجى مراجعة [تثبيت الأدوات](/ar/docs/tasks/tools/#kubectl) للحصول على تعليمات التثبيت.
+
+<!-- lessoncontent -->
+
+## إنشاء عنقود مينيكيوب
+
+```shell
+minikube start
+```
+
+## فتح  لوحة المعلومات
+
+افتح لوحة معلومات كوبيرنيتيس. يمكنك فعل ذلك بطريقتين مختلفتين:
+
+{{< tabs name="dashboard" >}}
+{{% tab name="Launch a browser" %}}
+افتح **سطر أوامر* جديد، وقم بتشغيل:
+
+```shell
+# ابدأ سطر أوامر جديد واترك هذا الأمر يعمل.
+minikube dashboard
+```
+
+الآن، قم بالتبديل إلى سطر الأوامر حيث قمت بتشغيل `minikube start`.
+
+## نشر التطبيق
+
+[*الحجيرة*](/ar/docs/concepts/workloads/pods/)كوبيرنيتيس هي مجموعة تتكون من حاوية واحدة أو عدة حاويات، مرتبطة معاً لأغراض الإدارة والشبكات. الحجيرة في هذا البرنامج التعليمي تحتوي على حاوية واحدة فقط. [*النشر*](/ar/docs/concepts/workloads/controllers/deployment/) في كوبيرنيتيس هو وحدة تتولى إنشاء وإدارة وتحجيم الحجيرات، وهي الأداة الموصى باستخدامها لتلك الأغراض. يحافظ النشر على صحة حجيرتك ويعيد تشغيل الحاويات داخلها إذا توقفت.
```

Comment:
```suggestion
[*الحجيرة*](/ar/docs/concepts/workloads/pods/) في كوبيرنيتيس هي مجموعة تتكون من حاوية واحدة أو عدة حاويات، مرتبطة معاً لأغراض الإدارة والشبكات. الحجيرة في هذا البرنامج التعليمي تحتوي على حاوية واحدة فقط. [*النشر*](/ar/docs/concepts/workloads/controllers/deployment/) في كوبيرنيتيس هو وحدة تتولى إنشاء وإدارة وتحجيم الحجيرات، وهي الأداة الموصى باستخدامها لتلك الأغراض. يحافظ النشر على صحة حجيرتك ويعيد تشغيل الحاويات داخلها إذا توقفت.
```

Reply:
I thought it was a typo a first but it looks like it is legit rectification

Reply:
Yeah thank you, it confused me at first too